package.json

```diff
@@ -19,6 +19,7 @@
     "husky": "^8.0.3",
     "i": "^0.3.7",
     "lint-staged": "^14.0.1",
+    "markdownlint": "^0.34.0",
```

Comment:
let's remove markdownlint as a dependency though on this and remove package-lock.json as this repo uses yarn not npm

Reply:
Do you mi d if we schedule a Google Meet ? This should help me understand better.

Reply:
I'll fix this up. All that is left is to please combine the new page you made with the existing docker page.

Reply:
> I'll fix this up. All that is left is to please combine the new page you made with the existing docker page.

All I am trying to understand is which existing page? Is the page on github? If yes post link then I can create a pull request. The link you dropped earlier directed me to a page outside github.